config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
Don't use the rocket syntax, always use the "new" hash syntax.
`controllers: { omniauth_callbacks: "users/omniauth_callbacks" }`